doc/2/guides/essentials/plugins/index.md

```diff
@@ -55,6 +55,13 @@ To install a plugin, you need to make it accessible in the `plugins/enabled` fol
 
 A common practice is to first copy the plugin to a `plugins/available` folder, and then creating a symbolic link from that folder to the `plugins/enabled` folder. This way, you can easily enable and disable a plugin just by creating or deleting a symbolic link, respectively.
 
+:::info
+<SinceBadge version="1.9.3" />
```

Comment:
```suggestion
<SinceBadge version="1.10.0" />
```
Since we are gonna release the new admin controller actions to encrypt/decrypt secrets